draft-ietf-ppm-dap.md

```diff
@@ -2121,6 +2128,54 @@ send an HTTP DELETE request to
 `{helper}/tasks/{task-id}/aggregation_jobs/{aggregation-job-id}` so that the
 Helper knows it can clean up its state.
 
+#### Batch Buckets {#batch-buckets}
+
+When aggregation recovers an output share, it must be stored into an appropriate
+"batch bucket", which is defined in this section. The data stored in a batch
+bucket is kept for eventual use in the {{collect-flow}}.
+
+Batch buckets are indexed by an aggregation parameter, as well as a batch
+mode-specific "batch bucket identifier":
+
+* For the time-interval batch mode, the batch bucket identifier is an interval
+  of time whose beginning is a multiple of the task's `time_precision` and whose
+  duration is equal to the task's `time_precision`. The identifier associated
+  with a given output share is the unique such interval containing the client
+  timestamp of the report the output share was recovered from.
```

Comment:
It might be helpful to give an example:
For example if `time_precision` is 3,600 seconds and the report's `timestamp` is `1729629081`, then the report's batch bucket is `...`